src/clj/game/core/engine.clj

```diff
@@ -1083,10 +1135,12 @@
   ([state _ eid] (checkpoint state nil eid nil))
   ([state _ eid {:keys [duration durations] :as args}]
    ;; a: Any ability that has met its condition creates the appropriate instances of itself and marks them as pending
-   (let [{:keys [handlers context-maps]} (mark-pending-abilities state eid args)]
+   (let [{:keys [handlers context-maps]} (mark-pending-abilities state eid nil)]
```

Comment:
Why are you not passing `args` to `mark-pending-abilities`?

Reply:
oops :joy: (it doesn't actually use them for anything ever I think - for some reason I thought it was nil before I tried a million different changes)